pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -142,6 +142,9 @@ func (mixerplugin) OnInboundCluster(env model.Environment, node model.Proxy, ser
 
 // OnOutboundRouteConfiguration implements the Plugin interface method.
 func (mixerplugin) OnOutboundRouteConfiguration(in *plugin.InputParams, routeConfiguration *xdsapi.RouteConfiguration) {
+	if in.Env.Mesh.MixerCheckServer == "" && in.Env.Mesh.MixerReportServer == "" {
```

Comment:
What will happen if only one of them is "" ?

Reply:
There is a way to disable checking only. Check cluster is not used if checking is disabled.

Reply:
I was under assumption if you have mixer enabled you probably want both policy and telemetry server, is  that not the case? also this check seems to be the pattern in `mixer.go`. https://github.com/istio/istio/pull/6636/files/de5e8548cd5f953e31d4342f3c5839bc1954154f#diff-d92228a902ebeed8f59f07f6408ae717R62 and  https://github.com/istio/istio/pull/6636/files/de5e8548cd5f953e31d4342f3c5839bc1954154f#diff-d92228a902ebeed8f59f07f6408ae717R92

Reply:
It used to be a single flag which is why it's AND. But there is no longer need to couple check/report, you can select whichever. The code was not updated to enable them selectively yet.